appservice/src/utils/workspace.ts

```diff
@@ -40,9 +40,9 @@ export async function selectWorkspaceFile(context: IActionContext, placeHolder:
 export async function selectWorkspaceItem(context: IActionContext, placeHolder: string, options: vscode.OpenDialogOptions): Promise<vscode.WorkspaceFolder | vscode.Uri> {
     let folder: vscode.WorkspaceFolder | undefined;
     if (vscode.workspace.workspaceFolders) {
-        const folderPicks: IAzureQuickPickItem<vscode.WorkspaceFolder | undefined>[] = await Promise.all(vscode.workspace.workspaceFolders.map((f: vscode.WorkspaceFolder) => {
+        const folderPicks: IAzureQuickPickItem<vscode.WorkspaceFolder | undefined>[] = vscode.workspace.workspaceFolders.map((f: vscode.WorkspaceFolder) => {
```

Comment:
Not really sure why this was a promise.all to begin with. I wonder if the map function used to have something async in it.